owncloudApp/src/main/java/com/owncloud/android/ui/activity/DrawerActivity.kt

```diff
@@ -0,0 +1,699 @@
+                    fetchIfNotCached = false,
+                    displayRadius = menuAccountAvatarRadiusDimension
+                )
+            }
+        }
+
+        // re-add add-account and manage-accounts
+        nav_view.menu.add(
+            R.id.drawer_menu_accounts, R.id.drawer_menu_account_add,
+            MENU_ORDER_ACCOUNT_FUNCTION,
+            resources.getString(R.string.prefs_add_account)
+        ).setIcon(R.drawable.ic_plus_grey)
+        nav_view.menu.add(
+            R.id.drawer_menu_accounts, R.id.drawer_menu_account_manage,
+            MENU_ORDER_ACCOUNT_FUNCTION,
+            resources.getString(R.string.drawer_manage_accounts)
+        ).setIcon(R.drawable.ic_group)
+
+        // adding sets menu group back to visible, so safety check and setting invisible
+        showMenu()
+    }
+
+    /**
+     * Updates the quota in the drawer
+     */
+    private fun updateQuota() {
+        Timber.d("Update Quota")
+        val account = AccountUtils.getCurrentOwnCloudAccount(this) ?: return
+        drawerViewModel.getStoredQuota(account.name)
+        drawerViewModel.userQuota.observe(this, Observer { event ->
+            when (event.peekContent()) {
+                is UIResult.Success -> {
+                    event.peekContent().getStoredData()?.let { userQuota ->
+                        when {
+                            userQuota.available < 0 -> { // Pending, unknown or unlimited free storage
+                                account_quota_bar?.visibility = View.VISIBLE
+                                account_quota_bar?.progress = 0
+                                account_quota_text?.text = String.format(
+                                    getString(R.string.drawer_unavailable_free_storage),
+                                    DisplayUtils.bytesToHumanReadable(userQuota.used, this)
+                                )
+                            }
+                            userQuota.available == 0L -> { // Quota 0, guest users
+                                account_quota_bar?.visibility = View.GONE
+                                account_quota_text?.text = getString(R.string.drawer_unavailable_used_storage)
+                            }
+                            else -> { // Limited quota
+                                account_quota_bar?.visibility = View.VISIBLE
+
+                                // Update progress bar rounding up to next int. Example: quota is 0.54 => 1
+                                account_quota_bar?.progress = ceil(userQuota.getRelative()).toInt()
+                                account_quota_text?.text = String.format(
+                                    getString(R.string.drawer_quota),
+                                    DisplayUtils.bytesToHumanReadable(userQuota.used, this),
+                                    DisplayUtils.bytesToHumanReadable(userQuota.getTotal(), this),
+                                    java.lang.String.valueOf(userQuota.getRelative())
```

Comment:
Use `import java.lang.String.valueOf` and write here only `valueOf(userQuota.getRelative())`